src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Apprentice/ApprenticeshipNotEnded.cshtml

```diff
@@ -0,0 +1,36 @@
+﻿@inject ILinkGenerator LinkGenerator
+@using SFA.DAS.CommitmentsV2.Shared.Extensions
+@using SFA.DAS.EmployerCommitmentsV2.Web.Extensions
+@using SFA.DAS.EmployerCommitmentsV2.Web.Models.Apprentice
+@model SFA.DAS.EmployerCommitmentsV2.Web.Models.Apprentice.ApprenticeshipNotEndedViewModel
+
+@{
+    ViewData["Title"] = "Apprenticeship has not yet completed";
+    ViewBag.Section = "apprentices";
+    ViewBag.PageId = "apprenticeship-not-ended";
+    ViewBag.GaData.Vpv = "/accounts/apprentices/manage/apprenticeship-not-ended";
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+        <h1 class="govuk-heading-xl">@ViewBag.Title</h1>
+
+        <p class="govuk-body">
+            You do not need to stop this apprenticeship;a completion payment still <br />
+            needs to be made to your training provider.Once this has been made the <br />
+            apprenticeship will complete automatically.<br />
+            <br />
+            For further information, contact your training provider.
+        </p>
+        <br />
+        <p class="govuk-body-s">
+            <a id="details-back-link" class="govuk-body govuk-link" href="@Url.Action("ApprenticeshipDetails", new ApprenticeshipDetailsRequest { AccountHashedId = Model.AccountHashedId, ApprenticeshipHashedId = Model.ApprenticeshipHashedId })">Go back to apprentice details</a>
+        </p>
```

Comment:
Can we simplify this markup, and remove all the `<br />`. Just use `<p>` tags, it keeps spacing consistent. e.g:
`        <p>
            You do not need to stop this apprenticeship; a completion payment still
            needs to be made to your training provider. Once this has been made the
            apprenticeship will complete automatically.
        </p>
        <p>
            For further information, contact your training provider.
        </p>
        <p>
            <a id="details-back-link" class="govuk-link" href="@Url.Action("ApprenticeshipDetails", new ApprenticeshipDetailsRequest { AccountHashedId = Model.AccountHashedId, ApprenticeshipHashedId = Model.ApprenticeshipHashedId })">Go back to apprentice details</a>
        </p>`